afinn/data/AFINN-no-165.txt

```diff
@@ -0,0 +1,3214 @@
+nerver	-1
+nervøs	-2
+nervøst	-2
+nei	-1
+ikke moro	-3
+edel	2
+edleste	2
+støyende	-1
+ikke-godkjent	-2
+nybegynner	-2
+ikke bra	-2
+jobber ikke	-3
+bemerkelsesverdig	2
+merkbar	2
+notorisk	-2
+roman	2
+nummen	-1
+pleie	2
+nøtter	-3
+utslette	-2
+utslettet	-2
+ytterst ubehagelig	-3
+obskøn	-2
+obskønitet	-2
+besatt	2
+foreldet	-2
+hindringer	-2
+sta	-2
+hinder	-2
+hindrer	-1
+merkelig	-2
+lovbrudd	-2
+fornærme	-2
+lovbryteren	-2
+uakseptable	-2
+støtende	-2
+offensivt	-2
+oks	2
+illevarslende	3
+en gang i livet	3
+muligheter	2
+mulighet	2
+undertrykt	-2
+undertrykkelse	-2
+undertrykkelser	-2
+undertrykkende	-2
+optimisme	2
+optimistisk	2
+utstøtt	-2
+au	-2
+utbrudd	-2
+ramaskrik	-2
+utmanøvrert	-2
+mindretall	-2
+opprør	-3
+oppsøk	2
+fremragende	5
+overbelastning	-1
+oversett	-1
+overbeskyttende	-2
+erobret	-2
+reagere	-2
+overreagerte	-2
+reagerer	-2
+overreaksjon	-2
+sell	-2
+sells	-2
+forglemmelse	-1
+forenkling	-2
+unyansert	-2
+overforenkle	-2
+solgt	-2
+overdriv	-2
+overvekt	-1
+overspent	-3
+smertelig	-2
+smertefull	-2
+panikk	-2
+fikk panikk	-3
+paradis	3
+paradoks	-1
+benådet	2
+tilgivelse	2
+benådninger	2
+lidenskap	1
+lidenskapelig	2
+passiv	-1
+passivt	-1
+patetisk	-2
+betale	-1
+fred	2
+fredelig	2
+straffe	-2
+straffet	-2
+straffer	-2
+penalising	-2
+straff	-2
+tankefull	-1
+perfekt	3
+perfeksjonert	2
+fullkommenhet	3
+perfeksjonerer	2
+peril	-2
+mened	-3
+begått	-2
+gjerningspersonen	-2
+gjerningsmenn	-2
+rådvill	-2
+forfølge	-2
+forfulgt	-2
+forfølger	-2
+opprørt	-2
+pervers	-3
+pessimisme	-2
+pessimistisk	-2
+forsteinet	-2
+filantropi	2
+fobiske	-2
+pittoreske	2
+hoper seg opp	-1
+plyndring	-2
+pirret	-2
+tiss	-4
+ynkelig	-2
+synd	-2
+medlidenhet	-2
+pest	-3
+leken	2
+vær så snill	1
+fornøyd	2
+behage	3
+gripende	2
+meningsløst	-2
+gift	-2
+forgiftet	-2
+forgifter	-2
+polert	2
+forurenseren	-2
+forurensere	-2
+forurensing	-2
+dårligere	-2
+fattigste	-2
+populær	3
+popularitet	3
+positiv	2
+positivt	2
+eiendomspronomen	-2
+post traumatisk	-2
+utsetter	-1
+fattigdom	-1
+kraftig	2
+maktesløs	-2
+ros	3
+skryt	3
+lovsanger	3
+priste	3
+be	1
+bønn	1
+ber	1
+rov	-2
+forberedt	1
+press	-1
+presset	-2
+late som	-1
+late	-1
+later	-1
+ganske	1
+forhindre	-1
+forhindret	-1
+stikk	-2
+fengsel	-2
+fange	-2
+fanger	-2
+privilegert	2
+proaktiv	2
+problemer	-2
+profitt	2
+lønnsom	2
+profitør	-2
+fortjeneste	2
+framgang	2
+forbyr	-1
+fremtredende	1
+love	1
+lovet	1
+løfter	1
+reklamere	1
+forfremmet	1
+fremmer	1
+fremme	1
+omgå	1
+tiltale	-1
+prospektet	1
+prospekter	1
+velstand	3
+velstående	3
+beskytte	1
+beskyttet	1
+beskytter	1
+demonstranter	-2
+protesterer	-2
+protester	-2
+stolt	2
+provosere	-1
+provosert	-1
+provoserer	-1
+vekk	-1
+klokskap	2
+pseudovitenskap	-3
+psykopatisk	-2
+reneste	1
+målbevisst	2
+skjelv	-2
+kvaliteter	2
+kvalitet	2
+avhør	-1
+rasisme	-3
+rasist	-3
+rasister	-3
+regn	-1
+tirade	-3
+ranters	-3
+voldtekt	-4
+voldtatt	-4
+voldtektsmann	-4
+henrykkelse	2
+opprykket	2
+begeistrede	4
+utslett	-2
+ratifisert	2
+å nå	1
```

Comment:
could you check this. Is "å" necessary here?

Reply:
I think this one should be ok actually. 
"nå" means just "now" and is too broad.
"å nå" means "to reach" and I've double checked in news and twitter search and is generally about reaching targets/milestones, reaching mount everest, reaching the youth.